src/main/scala/io/iohk/ethereum/jsonrpc/http/JsonRpcHttpServer.scala

```diff
@@ -31,10 +32,16 @@ class JsonRpcHttpServer(jsonRpcController: JsonRpcController, config: JsonRpcHtt
       }
       .result()
 
-  val route: Route =
+  val route: Route = {
     (pathEndOrSingleSlash & post & entity(as[JsonRpcRequest])) { request =>
       handleRequest(request)
+    } ~ post {
```

Comment:
in rejection we do not have the request, main purpose of this code is debugging, I can delete it